src/test/java/com/neuvector/ScannerTest.java

```diff
@@ -45,14 +57,30 @@ public void scanRegistryTest(){
         String repositoryTag = "3.6";
         String nvScannerImage = "neuvector/scanner:latest";
         String nvRegistryURL = registryURL;
-        String nvRegistryUser = "xxx";
-        String nvRegistryPassword = "xxx";
+        String nvRegistryUser = "";
+        String nvRegistryPassword = "";
         String mountPath = "/temp";  //mountPath is an optional parameter. It will use "/var/neuvector" by default.
 
         Registry registry = new Registry(registryURL, regUser, regPassword,repository,repositoryTag);
         NVScanner scanner = new NVScanner(nvScannerImage, nvRegistryURL, nvRegistryUser, nvRegistryPassword, mountPath);
 
         ScanRepoReportData scanReportData = Scanner.scanRegistry(registry, scanner, license);
+
+        File file = new File(mountPath + "/scan_result.json");
+        UserPrincipal user = getUserPrincipal(mountPath, file);
+
+        assertTrue(user.getName().equals("root") || user.getName().contains(System.getProperty("user.name")));
         assertTrue( scanReportData != null );
     }
+
+    private static UserPrincipal getUserPrincipal(String mountPath, File file) throws IOException {
+        UserPrincipal user = null;
+        if (file.exists()) {
```

Comment:
This if check here is not needed.. In tests `scan_result.json` file should always exist.

Reply:
efbf62144e1f51223c724be278c2a6dd0e6a99e9